packages/framework/src/Form/Admin/Order/OrderFormType.php

```diff
@@ -74,31 +81,136 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $domainId = $options['order']->getDomainId();
         $countries = $this->countryFacade->getAllByDomainId($domainId);
+        $builder
+            ->add($this->createBasicInformationGroup($builder, $options['order']))
+            ->add($this->createPersonalDataGroup($builder))
+            ->add($this->createCompanyDataGroup($builder))
+            ->add($this->createBillingDataGroup($builder, $countries))
+            ->add($this->createShippingAddressGroup($builder, $countries))
+            ->add($this->createNoteGroup($builder))
+            ->add('orderItems', OrderItemsType::class, [
+                'order' => $options['order'],
+            ])
+
+            ->add('save', SubmitType::class);
+    }
+
+    /**
+     * @param \Symfony\Component\OptionsResolver\OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setRequired('order')
+            ->setAllowedTypes('order', Order::class)
+            ->setDefaults([
+                'data_class' => OrderData::class,
+                'attr' => ['novalidate' => 'novalidate'],
+                'validation_groups' => function (FormInterface $form) {
+                    $validationGroups = [ValidationGroup::VALIDATION_GROUP_DEFAULT];
+
+                    $orderData = $form->getData();
+                    /* @var $data \Shopsys\FrameworkBundle\Model\Order\OrderData */
```

Comment:
Malformed PhpDoc for `@var` (see #555) and mistaken variable name, please fix to:
```php
/** @var \Shopsys\FrameworkBundle\Model\Order\OrderData $orderData */
$orderData = $form->getData();